### DISCUSSION TITLE
Discussing issues

### DISCUSSION THREAD

**Discussion by jatinkaushik-jk** [Avocodos/avocodos#13] [General]
# Discussing Issues

There's lot of issues I found today.

## **No realtime comment count updation**
![image](https://github.com/user-attachments/assets/4a83a7f9-7894-4c91-8c6c-8ad5631c2701)

## **Infinite loading time while pasting a text in post**

![image](https://github.com/user-attachments/assets/27b71773-f35c-4af3-b4b3-51962d6ba17e)

**Comment by HarjjotSinghh** [comment]
Commit b3fe310 summary: 

- The 1st issue has been _partially_ resolved. The comment count gets updated but it takes some time (around 5-10 seconds in development environment).

- The 2nd issue has been resolved compeletely.

- I am still keeping this discussion open until the 1st will be resolved completely.
